pandas/tests/frame/test_dtypes.py

```diff
@@ -21,6 +21,11 @@
 import pandas as pd
 
 
+@pytest.fixture(params=[str, compat.text_type])
```

Comment:
may consider in the future to move this fixture to conftest (needs a good name though)